server/src/config.ts

```diff
@@ -1,10 +1,29 @@
-/* tslint:disable:max-line-length */
+import { DbConfig } from "./models/index";
+
 export class Config {
   public static DB_USER: string = "postgres";
   public static DB_PW: string = "socobo";
-  public static DB_URL_TEST: string = `postgres://${Config.DB_USER}:${Config.DB_PW}@localhost:5432/socobo_test?ssl=false`;
-  public static DB_URL_DEV: string = `postgres://${Config.DB_USER}:${Config.DB_PW}@localhost:5432/socobo_dev?ssl=false`;
-  public static DB_URL: string = `postgres://${Config.DB_USER}:${Config.DB_PW}@localhost:5432/socobo?ssl=false`;
+  public static DB_CONFIG_TEST: DbConfig = {
```

Comment:
Hmm, I think yes. Otherwise we commit our real db connectionstring to github!? what if we implement a toString() function?